erpnext/projects/doctype/project/project.py

```diff
@@ -2139,3 +2150,31 @@ def set_warranty_claim_denied(projects, denied, reason=None):
 		doc.warranty_claim_denied = denied
 		doc.warranty_claim_denied_reason = reason
 		doc.save()
+
+
+@frappe.whitelist()
+def submit_feedback(project, customer_feedback):
+	if not customer_feedback:
+		frappe.throw(_('Remarks are mandatory for Communication'))
+
+	if not frappe.db.exists('Project', project):
+		frappe.throw(_("Repair Order {0} does not exist".format(project)))
+
+	target_doc = frappe.get_doc('Project', project)
+
+	cur_dt = get_datetime()
+	cur_date = getdate(cur_dt)
+	cur_time = get_time(cur_dt)
+
+	feedback_info = {
+		"feedback_date": cur_date,
+		"feedback_time": cur_time,
+		"customer_feedback": customer_feedback
+	}
+
+	target_doc.update(feedback_info)
+	target_doc.run_method("validate_feedback")
+
+	target_doc.db_set(feedback_info)
```

Comment:
Hmm... did you test this? I think it requires a 2nd positional argument which we usually pass as None